packages/build/bin/run-mocha.js

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env node
+// Copyright IBM Corp. 2017,2018. All Rights Reserved.
+// Node module: @loopback/build
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+/*
+========
+
+Usage:
+  node ./bin/run-mocha
+
+========
+*/
+
+'use strict';
+
+function run(argv, dryRun) {
+  const utils = require('./utils');
+  const path = require('path');
+
+  // Substitute the DIST variable with the dist folder
+  const dist = utils.getDistribution();
+  const mochaOpts = argv.slice(2).map(a => a.replace(/\bDIST\b/g, dist));
```

Comment:
Since this has effectively inlined `select-dist` script, I think we can remove `select-dist` entirely? Just make sure to mention that removal in the commit message as a BREAKING CHANGE.

Reply:
We have to leave `lb-dist` there to keep backward compatibility as projects generated from previous versions of our CLI reference `lb-dist`.

Reply:
Makes sense.

To me, this is another argument against using`.alpha.X` prereleases for so long. If we were versioning non-core modules as `0.x.y`, then we could treat semver-patch release as backwards compatible and semver-minor releases as breaking. This works great with the carrot operator as implemented by npm's semver (https://www.npmjs.com/package/semver#caret-ranges-123-025-004):

> this allows patch and minor updates for versions `1.0.0` and above, patch updates for versions `0.X >=0.1.0`, and no updates for versions `0.0.X`.

Reply:
Agree with the versioning. Is it too late to fix it?

Reply:
> Agree with the versioning.

👍 

> Is it too late to fix it?

I don't know. Let's open a new issue and try to find out what the implications of such change would be & how much work this change would require:

https://github.com/strongloop/loopback-next/issues/954